.gitmodules

```diff
@@ -91,4 +91,4 @@
 [submodule "platform/broadcom/saibcm-modules-dnx"]
 	path = platform/broadcom/saibcm-modules-dnx
 	url = https://github.com/Azure/saibcm-modules.git
-	branch =  sdk-6.5.22-gpl
+	branch =  sdk-6.5.22-gpl-dnx
```

Comment:
There are use cases that requires opennsl to be common across XGS and DNX. Do we really need to have separate kernel modules for DNX XGS?. Since its kernel module and only flags are passed to enable both XGS and DNX, can be it same regardless which SAI (user space driver)?.

Reply:
Based on the recommendation from Broadcom we kept it separate. In SAI release 5.0.0.* onwards the XGS ASIC family uses the BCM SDK 6.5.21 release, while the DNX asic family uses the BCM SDK 6.5.22 release ( as per Broadcom different asic families will go with different SDK releases in future as well)
Since there are new GPL module released, per SDK release + now that we use different SDK for different ASIC family - felt it is good to keep the kernel module separate.

Reply:
Since its kernel module and only thing i know of is, flags that required for supporting XGS and DNX, even older switches. for example, 6.5.22 kernel will have flags that will support XGS part of <= 6.5.22 SDK right. Can you check with broadcom and see if the flags can be added in kernel module in such a away, it would support different family of chipset as long as SDK is min version is met.

Reply:
Input from Broadcom : the fixes introduced in 6.5.22 GPL is tested with SDK 6.5.22, IF we need to do this Any fixes that are necessary in 6.5.22 GPL must be patched to GPL 6.5.21. 
We can check if this is feasible later - the core reasoning from Broadcom for this being the SDK used is different for XGS(6.521) and DNX(6.5.22) is different in same SAI(5.0) version.